client/src/components/layout/NewDogForm.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from "react"
+import ErrorList from "./ErrorList.js"
+import { Redirect } from "react-router-dom";
+import Fetcher from "./services/Fetcher.js";
+
+const NewDogForm = ({ user }) => {
+  const defaultFormValue = {
+    dogName: "",
+    pictureUrl: "",
+    description: "",
+    userId: user.id,
+  };
+
+  const [newDog, setNewDog] = useState(defaultFormValue);
+  const [errors, setErrors] = useState([]);
+  const [pictureUrl, setPictureUrl] = useState("");
```

Comment:
We shouldn't be adding placeholder images - uploading images is its own feature as it involves a couple new integrations. For the meantime, we should set up dogs to be created without an image until the group is ready to work on image uploading

Reply:
We aren't using this state to upload we have a "test url" button in the form that the user can use to test if the url they entered actually renders an image.